query-engine/connector-test-kit-rs/query-tests-setup/src/config.rs

```diff
@@ -27,34 +27,39 @@ impl Display for TestExecutor {
 pub struct TestConfig {
     /// The connector that tests should run for.
     /// Env key: `TEST_CONNECTOR`
-    connector: String,
+    pub(crate) connector: String,
 
     /// The connector version tests should run for.
     /// If the test connector is versioned, this option is required.
     /// Env key: `TEST_CONNECTOR_VERSION`
     #[serde(rename = "version")]
-    connector_version: Option<String>,
+    pub(crate) connector_version: Option<String>,
 
     /// An external process to execute the test queries and produced responses for assertion
     /// Used when testing driver adapters, this process is expected to be a javascript process
     /// loading the library engine (as a library, or WASM modules) and providing it with a
     /// driver adapter.
     /// Possible values: Napi, Wasm
     /// Env key: `EXTERNAL_TEST_EXECUTOR`
-    external_test_executor: Option<TestExecutor>,
+    pub(crate) external_test_executor: Option<TestExecutor>,
 
     /// The driver adapter to use when running tests, will be forwarded to the external test
     /// executor by setting the `DRIVER_ADAPTER` env var when spawning the executor process
-    driver_adapter: Option<String>,
+    pub(crate) driver_adapter: Option<String>,
 
     /// The driver adapter configuration to forward as a stringified JSON object to the external
     /// test executor by setting the `DRIVER_ADAPTER_CONFIG` env var when spawning the executor
-    driver_adapter_config: Option<serde_json::Value>,
+    pub(crate) driver_adapter_config: Option<DriverAdapterConfig>,
 
     /// Indicates whether or not the tests are running in CI context.
     /// Env key: `BUILDKITE`
     #[serde(default)]
-    is_ci: bool,
+    pub(crate) is_ci: bool,
```

Comment:
It looks like this is only true when `BUILDKITE` env var is set. But this runs on GitHub Actions too? Then this would be false, maybe that needs to be changed.